### ISSUE TITLE
[BUG] Fixed Footer does not respect Page Content Width

### ISSUE THREAD

**Issue by pholst** [kirbydesign/designsystem#3729]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Fixed Footer on Kirby Page does not seem to respect the Page Content Width set with `maxWidth` on Kirby Page.
 

## Describe how to reproduce the bug
The bug can easily be reproduced with the following snippet:
```
<kirby-page [maxWidth]="'lg'">
  <kirby-page-content>
    // Content width will be Large, 1092px
  </kirby-page-content>
  <kirby-page-footer>
    // Content width will be Default, 792px
  </kirby-page-footer>
</kirby-page>
```

## Which Kirby version was used?
10.1.3 

## What was the expected behavior?

It is expected that the content width of Kirby Page Footer is the same as Kirby Page Content, i.e. Large, 1092px;

## Add any screenshots
![image](https://github.com/user-attachments/assets/b61cc8df-beab-4b12-95fb-82beca9b969b)

## Please complete the following information:
The bug was registered with the following setup, but is expected to be consistent across all platforms/browsers:
- OS: [Windows]
- Browser [Edge]
- Browser version [130.0.2849.99]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!

**Comment by alxzak** [comment]
Possible workaround: 
@use '@kirbydesign/designsystem/scss/utils' as *;

kirby-page-footer {
 --page-content-max-width: #{get-page-content-max-width('lg')};
}
